.github/workflows/build.yml

```diff
@@ -103,12 +113,19 @@ jobs:
           cp ./arm64-osx.cmake ./community/universal-osx.cmake
           sed -i '' 's/^set(VCPKG_OSX_ARCHITECTURES.*$/set(VCPKG_OSX_ARCHITECTURES "arm64;x86_64")/' ./community/universal-osx.cmake
 
+      - name: Import macOS code signing certificate
+        if: ${{ matrix.os.preset == 'mac' && env.MACOS_SIGNING == 'true' }}
+        uses: apple-actions/import-codesign-certs@v3
```

Comment:
worth noting just for diligence reasons: `apple-actions` appears to be a non-Apple-owned repository, and compromise of the actions coming from that repository (and thus compromise of our certificates) is something we should consider in our threat model. in the long term, it might be better to fork that repository into one controlled exclusively by @DescentDevelopers/d3-maintainers and use that.

Reply:
Fixed.